inspirehep/modules/records/receivers.py

```diff
@@ -170,20 +175,106 @@ def populate_bookautocomplete(sender, json, *args, **kwargs):
     input_values.extend(titles)
     input_values = [el for el in input_values if el]
 
-    ref = get_value(json, 'self.$ref')
+    record = get_value(json, 'self.$ref', '')
 
     json.update({
         'bookautocomplete': {
             'input': input_values,
             'payload': {
                 'authors': authors,
-                'id': ref,
+                'id': record,
                 'title': titles,
             },
         },
     })
 
 
+def populate_book_series_suggest(sender, json, *args, **kwargs):
+    """Populate the ``book_series.book_series_suggest`` field of Literature records."""
+    if 'hep.json' not in json.get('$schema'):
```

Comment:
is this really one should check for a given record type (cc @jacquerie)? I would expect there to be a helper somewhere that gets you the type from the record. If not, it should probably exist.

Reply:
@michamos is this related: https://github.com/inspirehep/inspire-next/issues/1355 ?